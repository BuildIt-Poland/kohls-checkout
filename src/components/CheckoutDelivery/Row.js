// Responsive form row wrapper

import styled from 'styled-components';

import { spacingLarge } from '../../styles/designTokens';
import media from '../../styles/media';

const Row = styled.div`
  display: grid;
  grid-template-columns: 1fr 1fr;
  column-gap: ${spacingLarge};

  ${media.phone`
    display: block;
  `};
`;

export default Row;
