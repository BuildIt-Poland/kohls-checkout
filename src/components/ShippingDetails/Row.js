import styled from 'styled-components';

import Text from '../Text';

const Row = styled(Text)`
  margin: 0;
`;

export default Row;
