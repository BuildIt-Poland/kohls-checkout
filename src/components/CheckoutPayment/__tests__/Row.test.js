import React from 'react';
import { create } from 'react-test-renderer';

import Row from '../Row';

describe('COMPONENT - CheckoutPayment Row', () => {
  it('renders correctly', () => {
    const component = create(<Row />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
