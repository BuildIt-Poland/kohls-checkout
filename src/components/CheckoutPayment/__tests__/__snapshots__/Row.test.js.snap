// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - CheckoutPayment Row renders correctly 1`] = `
.c0 {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  -webkit-column-gap: 3rem;
  column-gap: 3rem;
}

@media (max-width:26.25em) {
  .c0 {
    display: block;
  }
}

<div
  className="c0"
/>
`;
